packages/patternfly-react/src/components/SessionTimeout/helpers.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Session extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      now: this.props.now(),
+      expiresAt: this.props.expiresAt
+    };
+  }
+
+  componentWillMount() {
+    this._interval = setInterval(
+      () => this.setState(() => ({ now: this.props.now() })),
+      1000
+    );
+  }
+
+  componentWillUnmount() {
+    clearInterval(this._interval);
+  }
+
+  render() {
+    const logout = () => {
```

Comment:
These can be class methods, but can also be removed if the component is used to trigger call backs instead of rendering.